spec/helpers/leagues_helper_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe LeaguesHelper, type: :helper do
+  describe "#countries" do
+    def get_leagues(country_code: nil)
```

Comment:
not sure defining methods in specs is a sound approach. we should review this on a 1:1 video call